src/sync.ts

```diff
@@ -138,7 +147,6 @@ function getPropertiesFromIssue(issue: Issue): CustomValueMap {
     milestone,
     created_at,
     updated_at,
-    body,
```

Comment:
Can we keep the body field as well? People might want to use it as a filter in the table

Reply:
Sure, do we want to display everything or only something like the first lines?
Because if we display everything then the page will have to copies of the same body